README.md

```diff
@@ -1,70 +1,51 @@
-Resources are prepared to enable running workflows on the cloud either
-doing manual interaction with Cromwell, or leveraging GMS to automate
-interactions through its familiar interface.
+# Cloud Workflow Orchestration
 
+This repo contains resources designed to enable running genomic analysis workflows on the cloud. This includes documentation and tools for setting up Google Cloud appropriately, spinning up VMs to run the workflow, monitoring runs, and retrieving results.
 
-# Motivation
+It is designed to be used in concert with the workflows in the [analysis-wdls repository](https://github.com/wustl-oncology/analysis-wdls), though other WDL workflows should certainly work as well.  
 
-Switching workflow runs to the cloud are primarily motivated by issues
-of reliability with the local cluster. Secondary motivations are
-numerous, ranging from scalability to data-sharing.
+## Quick Start
+There are two supported methods of launching workflows documented in the following pages:
 
+1. [manual submission to Cromwell](https://github.com/wustl-oncology/cloud-workflows/blob/main/manual-workflows/README.md), 
+2. leveraging WUSTL's [GMS to automate interactions](https://github.com/wustl-oncology/cloud-workflows/tree/main/gms).
 
-# When to Use
+There are also several end-to-end demonstrations of running our WDL workflows on Google Cloud:
 
-We're starting with a few known use cases that we believe would
-benefit from switching to the cloud.
-
-The two use cases we're aiming at first are
-1) ad-hoc workflow kickoff through manual intervention
-2) workflows kicked off through the GMS
-
-
-## Manual Kickoff
-
-For people who want to fiddle with workflows, or otherwise dive into
-the gears of Cromwell without the burden of the extra GMS
-abstractions, this is the approach.
+- Running the Immunogenomics workflow (immuno.wdl) manually and 
+assuming that your input data is stored on the compute1/storage1 
+system at WASHU: [immuno-compute1](https://github.com/griffithlab/immuno_gcp_wdl_compute1)  
 
-For more information on manual interactions, see
-[manual-workflows/README.md](manual-workflows/README.md)
+- Running the Immunogenomics workflow (immuno.wdl) manually and
+assuming the your input data is on a local machine that is not 
+associated with WASHU in any way. For example, an external institution
```

Comment:
The style guide says to use `WashU` with only the `W` and `U` capitalized: https://marcomm.wustl.edu/items/washington-university-in-st-louis-washu/  Annoyingly it also says not to use `WUSTL` any more.
```suggestion
associated with WashU in any way. For example, an external institution
```